css/main.scss

```diff
@@ -0,0 +1,110 @@
+/* Define standard variables and values for website */
+$bgcolor: #ff4dff;
```

Comment:
I like the use of variables. However, I would like us to only use CSS not SCSS for our projects.
See this [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#basic_usage) for more information on CSS variables.

Reply:
According to the learning video, the main.scss generates the .css file. 

.scss has easier/simpler syntax. I can add the .scss files to git ignore. I think it is unnecessary to push them.

Reply:
Switched to using custom property.

<img width="451" alt="image" src="https://user-images.githubusercontent.com/35523688/225400183-31679aae-9c43-46fa-aa80-9dfe75e13931.png">